doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,290 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
 
-#### Creating a publisher
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
 
-When creating an untyped publisher we do not need to specify a data type as template paraemter.
+A triple consisting of such strings is called a ``ServiceDescription``. Two ``ServiceDescription``s are considered 
+matching if all these three strings are element-wise equal, i.e. group, instance and topic names are the same for both 
+of them. This means the group and instance identifier can be ignored to create different ``ServiceDescription``s. They 
+will be needed for advanced filtering functionality in the future.
 
-```cpp
-iox::popo::UntypedPublisher publisher({"Group", "Instance", "CounterTopic"});
-publisher.offer();
-```
+The service model of iceoryx is derived from AUTOSAR and is still used in the API with these names. The so called 
+canonical protocol is implemented in the namespace ``capro``.
 
-#### Sending data
+The following table gives an overview of the different terminologies and the current mapping:
 
-##### 1. Loan and emplace
+|         | Group   | Instance         | Topic                  |
+|---------|---------|------------------|------------------------|
+| ROS2    | Type    | Namespace::Topic | -                      |
+| AUTOSAR | Service | Instance         | Event                  |
+| DDS     | -       | -                | /Group/Instance/Topic  |
 
-Before sending, we have to loan a chunk of memory to emplace our data.
+Service is related to instance like classes are related to objects in C++. Service describes an abstract topic and an 
+instance is one instantiation of that abstraction. Like an object is an instantiated class. Events are in this context 
+like members of a class.
 
+Example:
 ```cpp
-auto result = publisher.loan(sizeof(CounterTopic));
+class MyRadarService {
+   public:
+      bool hasObstacleDetected;
+      float distanceToObstacle;
+};
+
+MyRadarService frontLeftRadarInstance;
+std::cout << frontLeftRadarInstance.hasObstacleDetected << std::endl;
 ```
+In the iceoryx world we would subscribe to the service ``("MyRadarService", "frontLeftRadarInstance", "hasObstacleDetected")`` 
+and would receive a sample whenever an obstacle was detected. Or we would subscribe to ``distanceToObstacle`` and would 
+receive a constant stream of data which presents the distance to the obstacle.
 
-Since the data type is not known to the publisher, we have to provide the size in bytes of the payload data we
-intend to send.
+#### Restrictions
 
-If we successfully acquired a chunk, we can construct the data to be send using placement new and publish it.
-Notice that ``sample.get()`` returns a ``void*`` to the memory ``chunk`` were we can then place our data.
-Depending on what we want to send, we may also use ``memcpy`` to copy the data to the ``chunk``.
+The data type of the transmitted data can be any C++ class, struct or plain old data type as long as it satisfies the 
+following conditions:
+- no heap is used
```

Comment:
Please add and additional line break between line 202 / 203. Otherwise MkDocs does not detect is as an unordered list.